src/icalendar/tests/prop/test_vWeekday.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from icalendar.prop import vWeekday
+
+
+def test_simple():
+    weekday = vWeekday("SU")
```

Comment:
I like these examples from the tests.
We also have doctests.
If you add some examples to class definition, this will help people understand your fix and how to use this class.

Like:
```
>>> su = vWeekday("SU") # Sunday
>>> su
>>> su.weekday
>>> su.relative
>>> su.to_ical()

>>> th = vWeekday("2TH")  # second Thursday of the month?
>>> th
>>> th.weekday
>>> th.relative
>>> th.to_ical()

>>> fr = vWeekday("-1FR")  # last Friday of the month?
>>> fr
>>> fr.weekday
>>> fr.relative
```

I am writing this here because I would like your input: What does -1 mean in your view? Is it the last weekday of the month?

Reply:
Yes. For example, this rrule would repeat monthly on every last friday: `FREQ=MONTHLY;BYDAY=-1FR`. I think using a relative BYDAY value is also allowed with a yearly frequency ([rfc](https://www.rfc-editor.org/rfc/rfc5545.txt), page 42)

Reply:
It might be nice to extend the rrule documentation, too. I would say though that I open another issue for that.

Reply:
I have updated the docstring and added some doctests now :)